samples/AzureFunctionsApp/AzureFunctionsApp.csproj

```diff
@@ -1,4 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
nitpick: should we remove this empty change?

Reply:
This is intentional - adding a new line for formatting of the document.

Reply:
Oh, interesting. So this is required as per some linter? Anyways, if it's intentional, it's fine

Reply:
Not required, my own preference.